account_statement_import_camt/models/account_statement_import.py

```diff
@@ -12,6 +12,7 @@
 class AccountBankStatementImport(models.TransientModel):
     _inherit = "account.statement.import"
 
+    # pylint: disable=W7938
```

Comment:
thanks for adding the `disable=`. However, can you add it on line 33 directly (or 34, not sure what is needed)? And I have a slight preference for the human readable form `except-pass`.

Reply:
I didn't know I can use `pylint: disable=except-pass` :smile:
Now it's clear for me what people are talking about by "human-readable" message